File: curso-python/ex029.py
# escreva um programa que leia a velocidade de um carro
vel = float(input("velocidade do carro(km):"))
# se ele ultrapassar 80km/h, mostre uma mensagem dizendo que ele foi multado
# a multa vai custar R$7,00 por cada km acima do limite
if vel > 80:
    excesso = vel - 80
    multa = 7 * excesso
    print("você foi multado")
    print(f"a sua multa vai custa R${multa}")
else:
    print("Você está na velocidade correta. Continue assim!")

File: javascript-exercicios/ex029.js
const prompt = require("prompt-sync")();

const vel = parseFloat(prompt("Velocidade do carro (km/h): "));

if (vel > 80) {
    const excesso = vel - 80;
    const multa = excesso * 7;
    console.log("Você foi multado!");
    console.log(`A sua multa vai custar R$${multa.toFixed(2)}`);
} else {
    console.log("Você está na velocidade correta. Continue assim!");
}
